handler.go

```diff
@@ -145,7 +145,7 @@ func (handler *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 		if r.Method == "OPTIONS" {
 			// Preflight request
-			header.Set("Access-Control-Allow-Methods", "POST, HEAD, PATCH, OPTIONS")
+			header.Set("Access-Control-Allow-Methods", "POST, GET, HEAD, PATCH, DELETE")
```

Comment:
Why did you remote the OPTIONS method? A client may send a OPTIONS request to determinate the server's configuration.